omnigibson/prims/material_prim.py

```diff
@@ -568,15 +568,18 @@ def emissive_color(self):
         Returns:
             3-array: this material's applied (R,G,B) emissive_color
         """
-        return np.array(self.get_input(inp="emissive_color"))
+        color = self.get_input(inp="emissive_color")
+        if color is None:
+            return None
+        return th.tensor(color)
```

Comment:
good catch, what about all the other properties in this class? Would they return `None` as well?